ggshield/scan/docker.py

```diff
@@ -148,13 +166,11 @@ def _get_layer_files(archive: tarfile.TarFile, layer_info: Dict) -> Iterable[Fil
         if not file_info.isfile():
             continue
 
-        if file_info.size > MAX_FILE_SIZE * 0.95:
-            continue
```

Comment:
I think this test should be brought back, to avoid scanning large files.

Reply:
Oh good catch, but then I think it's better to remove the other line, because the first check does not read the file content, whereas the second does.

Reply:
Hmm we have to read it anyway for the `File.from_bytes` call, no ? 🤔

Reply:
Yes, but the early check avoid reading the content of large files we are going to ignore.

Reply:
Alright, should be better now. Thanks.